src/Accounts/Accounts/Rest/InvokeAzRestMethodCommand.cs

```diff
@@ -111,91 +125,309 @@ public override void ExecuteCmdlet()
                 this.Path = Uri.PathAndQuery;
             }
 
-            IAzureRestClient serviceClient = null;
-            if (ByPath.Equals(this.ParameterSetName) || ByParameters.Equals(this.ParameterSetName))
+            IAzureRestClient serviceClient = this.InitializeServiceClient();
+
+            AzureOperationResponse<string> response = ExecuteRestRequest(serviceClient);
+            WriteObject(new PSHttpResponse(response));
+
+            if (WaitTillCompletion.IsPresent && IsRequestLRO(response))
```

Comment:
(High) It's an interestring scenario to think about when `-WaitTillCompletion` is present but the request is not actually an LRO. Do we want to throw an error? Explain why we don't think it's an LRO? Or simply fall back to the original behavior i.e. returning the first response?

1. Consider adding a warning message saying that you shouldn't use -WaitTillCompletion
2.

Reply:
Added this warning message.